### DISCUSSION TITLE
Issue with Custom Resource Handler

### DISCUSSION THREAD

**Discussion by rvjykmr** [cefsharp/CefSharp#4335] [General]
I have written a custom resource handler by deriving from ResourceHander class and overriding the ProcessRequestAsync method.

The intention is to execute custom code to launch a windows executable on click of a anchor tag having href matching the scheme name .
But I notice the browser re-direct to the blank page once this ProcessRequestAsync is completed.

I was returning CefReturnValue.ContinueAsync from the overridden ProcessRequestAsync method.

Is there any way to avoid the browser control not to redirect to blank page?

**Comment by amaitland** [comment]
> The intention is to execute custom code to launch a windows executable on click of a anchor tag having href matching the scheme name .
But I notice the browser re-direct to the blank page once this ProcessRequestAsync is completed.

If you don't populate the response stream then a blank page is expected. 

There's an example of the approach I'd recommend at https://github.com/cefsharp/CefSharp/wiki/General-Usage#protocol-execution
